redaxo/src/addons/backup/lib/cronjob.php

```diff
@@ -28,9 +28,24 @@ public function execute()
             }
             $file = $file . '_' . $i;
         }
+        $exportFilePath = $dir . $file . $ext;
 
-        if (rex_backup::exportDb($dir . $file . $ext, $whitelist_tables)) {
-            $message = $file . $ext . ' created';
+        if (rex_backup::exportDb($exportFilePath, $whitelist_tables)) {
+            $message = basename($exportFilePath) . ' created';
+
+            if ($this->getParam('compress')) {
+                $archiveFilePath = $exportFilePath.'.tar.gz';
```

Comment:
Da es nur eine Datei ist, würde ich das tar-Archiv eigentlich gerne weglassen, und die Datei direkt gz-komprimieren.
Also so in der Art: https://stackoverflow.com/a/22754032/3837585
Am Ende sollte es aus meiner Sicht eine `.sql.gz` sein.

Reply:
Da die aktuelle implementierung auf existierenden apis beruht sehe ich keinen vorteil es zu ändern?

Reply:
Es entspricht mehr dem Standard, andere Tools exportieren auch `.sql.gz`-Dateien, und können diese auch direkt importieren.
So zum Beispiel in Sequel Ace, was ich für MySQL nutze.
Ich glaube aber, phpmyadmin kann es auch.

Reply:
PR und beschreibung angepasst.